ote/src/clj/ote/services/taxiui_service.clj

```diff
@@ -90,17 +90,27 @@
               (some->> (:unknown-places places)
                        (log/warnf "Detected unknown places while updating %s/%s details! %s" operator-id service-id)))))))))
 
+(defn- age-filter
+  "Returns age filter as PostgreSQL INTERVAL compatible string."
+  [filters]
+  (case (:age-filter filters)
+    :within-six-months "6 months"
+    :within-one-year   "11 months 27 days"  ; this is a minor hack to allow easy "diff >= 1 year" selection in SQL
```

Comment:
Does this hack affect the functionality of the application (selection within 1 year)?

Reply:
In a positive way, yes. The problem is expressing "less than 1 year" in same CASE in the SQL query as "more than 1 year" with a single boolean and a simple clause, and I didn't want to overcomplicate the query.